code/__DEFINES/shuttles.dm

```diff
@@ -52,12 +52,12 @@
 #define ENGINE_DEFAULT_MAXSPEED_ENGINES 5
 
 //Docking error flags
-#define DOCKING_SUCCESS 0
-#define DOCKING_BLOCKED 1
-#define DOCKING_IMMOBILIZED 2
-#define DOCKING_AREA_EMPTY 4
-#define DOCKING_NULL_DESTINATION 8
-#define DOCKING_NULL_SOURCE 16
```

Comment:
hmm, I wonder if these bitflags were used as bitflags...

Reply:
nope

Reply:
ninja says they aren't used as bitflags

Reply:
ninja = guy who coded it

Reply:
apparently he's wrong though

Reply:
They aren't really used as bitflags but I have them like this so they can be used like that if need be, I did at one part treat them like a bitflag but it'd be extremely easy to change that piece of code since there shouldn't be any point where there are actually multiple of these in a single var

Reply:
I'd prefer we keep its ability to be a bitflag if need be later when/if I ever feel like making it possible for multiple errors to be detected at once.